lib/betagouvbot/format_mail.rb

```diff
@@ -0,0 +1,89 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+require 'kramdown'
+
+module BetaGouvBot
+  class FormatMail < Hash
+    class << self
+      def to_rules
```

Comment:
When I read "to_x" I expect that something (a value) is being converted to a different type or representation. Here we have a class, not a value, and "rules" does not denote a type or representation, so this violates my expectations for understanding.

And, I'm sorry, I still don't feel like rules belong in this class. We format a number of other kinds of mails that have nothing to do with these time horizons, so this class has too many distinct reasons to change; my sense of where the rules belong is more in the Anticipator class (because they're the responses to the time horizons that Anticipator consumes as input data). The principle at work here would be to keep data close to where it's being used.